### DISCUSSION TITLE
Id-Error when placeholder is added to ListElement-Component

### DISCUSSION THREAD

**Discussion by stevan06v** [livewire/livewire#8123] [1. Help]
I am using volt and I am looping through a list of values and displaying the component(i added the id's):

**My Components:**
-  Card
-  List

**Card**

```
new class extends Component {
    public function placeholder()
    {
        return <<<'HTML'
        <center class="p-10">
            <sl-spinner class="text-7xl" style="--track-width: 9px;"></sl-spinner>
        </center>
        HTML;
    }
};
```

**List**

```
@foreach($projects as $project)
            <div class="mx-8 border-other-grey border-2 rounded-2xl mt-6" wire:key="{{$project->id}}">
                <livewire:projects.card lazy="true" :project="$project" :key="$project->id"/>
            </div>
@endforeach
```

When I remove the placeholder function I do not encounter the error below.

How do I fix that? I fetch data so I have to show the user some response.

**The Error:**
Uncaught Snapshot missing on Livewire component with id: uRmvWvURv2IivMsAh9Ug

**Comment by joshhanley** [comment]
@Stevan06v what does your whole card component look like?
